plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineDeployInfoTest.java

```diff
@@ -0,0 +1,71 @@
+package com.google.cloud.tools.eclipse.appengine.deploy;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.eclipse.core.runtime.CoreException;
+import org.junit.Test;
+import org.xml.sax.SAXException;
+
+public class AppEngineDeployInfoTest {
+
+  private static final String TEST_VERSION = "fooVersion";
+  private static final String TEST_ID = "fooId";
+
+  private static final String XML_SUFFIX = "</appengine-web-app>";
+  private static final String XML_DECLARATION = "<?xml version=\"1.0\" encoding=\"utf-8\"?>"
```

Comment:
you can use single quotes instead of double quotes to avoid escaping and make it a little more readable